site24x7/websitemonitor.go

```diff
@@ -1,199 +1,230 @@
 package site24x7
 
 import (
-	"bytes"
-	"encoding/json"
-	"fmt"
-	"log"
-	"net/http"
 	"strconv"
 
+	site24x7 "github.com/Bonial-International-GmbH/site24x7-go"
+	"github.com/Bonial-International-GmbH/site24x7-go/api"
+	apierrors "github.com/Bonial-International-GmbH/site24x7-go/api/errors"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 )
 
+var WebsiteMonitorSchema = map[string]*schema.Schema{
+
+	"display_name": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+
+	"website": {
```

Comment:
Can you remove the empty lines between the map items please? :)

Reply:
Done